gui/test/integration/cypress/support/page_files/NodeSearchInput.ts

```diff
@@ -0,0 +1,102 @@
+class NodeSearchSuggestionLink {
+  constructor(
+    private readonly index: number,
+    private readonly searchSuggestions: NodeSearchSuggestions
+  ) {}
+
+  get() {
+    return this.searchSuggestions.get().find("ul>a").eq(this.index);
```

Comment:
Prefer data-cy if possible, please.

Reply:
Not sure if I have access to index with new setup